plugins/oracle/oracle_sysstat_

```diff
@@ -531,7 +678,7 @@ getvalue_sysstat() {
     do
         [ -z "$field" ] && continue
 
-        ${sqlplus} "${oracle_auth}" <<EOF
+        cat <<EOF
```

Comment:
Instead of `cat <<EOF` you could also use a simple echo statement. In this case it would not even interfere with the quoting.